.air.toml

```diff
@@ -0,0 +1,16 @@
+# Config file for [Air](https://github.com/cosmtrek/air) in TOML format
```

Comment:
Is the live reloading important for the API, or is that just your personal preference for development? My vote would be to add it to .gitignore instead, mainly because we have to have a different build system that we are actually being graded on (building the run executable, using `./run build` to build source code, etc.). Updating two separate build systems could be confusing

Reply:
i made these just in case someone's configurations would use these, as I was using a boilerplate tutorial that supplied them.

Reply:
I think we should leave this part out of the main branch then. Since the Autograder API specifies way we should compile/run our code, I think our main branch should only use the Autograder method. That way, we can know if our code runs correctly when compiled the way it will under the Autograder.

What if we add your alternative build system to a permanent branch? That way, people who want to use that method can still access it.

Reply:
I think we can just toss it, it looks like an artifact from my initial design failures, and prob isn't relevant